05-Organizing-and-Executing-Simulations/01-Organizing-Data.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>We will likely be removing the capability to name the workspace. We should just use <code>workspace</code> since this is likely to change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
That part was more a comment that this will need to change in the future to keep up with the modern versions.